websocket-integration/src/main.rs

```diff
@@ -0,0 +1,106 @@
+mod messages;
+mod service;
+mod wshandler;
+
+use crate::wshandler::WsHandler;
+
+use dotenv::dotenv;
+
+use actix_web::web::Payload;
+use actix_web::{get, web, App, Error, HttpRequest, HttpResponse, HttpServer};
+use actix_web_actors::ws;
+
+//use actix_web::middleware::Condition;
+//use actix_web_httpauth::extractors::bearer::BearerAuth;
+
+use drogue_cloud_service_common::{
+    config::ConfigFromEnv, health::HealthServer, openid::Authenticator, openid_auth,
+};
+use drogue_cloud_service_common::{defaults, health::HealthServerConfig};
+use serde::Deserialize;
+
+use crate::service::Service;
+use actix::{Actor, Addr};
+use futures::TryFutureExt;
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Config {
+    #[serde(default = "defaults::bind_addr")]
+    pub bind_addr: String,
+    #[serde(default = "defaults::enable_auth")]
+    pub enable_auth: bool,
+
+    #[serde(default)]
+    pub health: HealthServerConfig,
+}
+
+#[get("/stream/{application}")]
+pub async fn start_connection(
+    req: HttpRequest,
+    stream: Payload,
+    application: web::Path<String>,
+    service_address: web::Data<Addr<Service>>,
+) -> Result<HttpResponse, Error> {
+    let application = application.into_inner();
+
+    let ws = WsHandler::new(application, service_address.get_ref().clone());
+    let resp = ws::start(ws, &req, stream)?;
+    Ok(resp)
+}
+
+#[actix_web::main]
+async fn main() -> anyhow::Result<()> {
+    env_logger::init();
+    dotenv().ok();
+
+    log::info!("Starting WebSocket integration service endpoint");
+
+    // Initialize config from environment variables
+    let config = Config::from_env().unwrap();
+
+    let enable_auth = config.enable_auth;
+
+    let authenticator = if enable_auth {
+        Some(Authenticator::new().await?)
+    } else {
+        None
+    };
```

Comment:
@ctron I am not sure if that is enough, as I found different things when looking at the other service, This looked like the minimal viable thing :)